docs/manual/scala/guide/cluster/code/docs/home/scaladsl/persistence/BlogState.scala

```diff
@@ -6,7 +6,10 @@ object BlogState {
   val empty = BlogState(None, published = false)
 
   //FIXME serialization, how to handle Option?
```

Comment:
Needs an import and a dependency on the playJson module from also to actually compile. Working on that now.